Mathlib/Analysis/Calculus/FDeriv/Analytic.lean

```diff
@@ -294,3 +296,71 @@ theorem CPolynomialOn.iterated_deriv (h : CPolynomialOn 𝕜 f s) (n : ℕ) :
   · simpa only [Function.iterate_succ', Function.comp_apply] using IH.deriv
 
 end deriv
+
+namespace FormalMultilinearSeries
+
+variable (p : FormalMultilinearSeries 𝕜 E F)
+
+/-- This series appears in `HasFPowerSeriesOnBall.fderiv` -/
+noncomputable
+def derivSeries : FormalMultilinearSeries 𝕜 E (E →L[𝕜] F) :=
+  (continuousMultilinearCurryFin1 𝕜 E F : (E[×1]→L[𝕜] F) →L[𝕜] E →L[𝕜] F)
+    |>.compFormalMultilinearSeries (p.changeOriginSeries 1)
```

Comment:
Now that we've made this a def, we should probably use this def in statements (for example `HasFPowerSeriesOnBall.fderiv`) as much as possible.

Reply:
But maybe not in this PR...

Reply:
There are just three occurrences:
`HasFPowerSeriesOnBall.fderiv`
`HasFiniteFPowerSeriesOnBall.fderiv`
`HasFiniteFPowerSeriesOnBall.fderiv'`
where the long expression can be replaced by `p.derivSeries`.

Reply:
OK.